cmd/mvn.go

```diff
@@ -0,0 +1,44 @@
+/*
+
+Copyright (c) nexB Inc. and others. All rights reserved.
+ScanCode is a trademark of nexB Inc.
+SPDX-License-Identifier: Apache-2.0
+See http://www.apache.org/licenses/LICENSE-2.0 for the license text.
+See https://github.com/aboutcode-org/dependency-inspector for support or download.
+See https://aboutcode.org for more information about nexB OSS projects.
+
+*/
+
+package cmd
+
+import (
+	"github.com/aboutcode-org/dependency-inspector/internal"
+	"github.com/spf13/cobra"
+)
+
+func mvnCmd() *cobra.Command {
+	lockFiles := []string{"lockfile.json"} // Question for Code Review: Is this sufficient? Might overwrite lockfile.json in submodule
+	lockGenCommand := []string{"mvn", "io.github.chains-project:maven-lockfile:generate"}
```

Comment:
Does this require any change to POM or the environment? Or will this automatically install that plugin? 

If yes, can this plugin be pinned to a specific version? 
Which Maven repo would used to fetch it? 

All these questions are to find out if we have minimum dependencies and moving parts to generate the lockfile.

Reply:
My comment disappeared from this thread as well.

It is a very good idea to pin it, especially since using an older version will not respect the `-DlockfileName` parameter. I have pinned this to version 5.3.5 using `mvn io.github.chains-project:maven-lockfile:5.3.5:generate`.

No changes are required to the pom, the `mvn` command will automatically install the plugin and all required dependencies.

Both maven-lockfile and all its dependencies are hosted on maven-central repository.